README.md

```diff
@@ -91,3 +91,15 @@ fields. To switch off this output, change output_convergence from True
 to False in the Python test definition. Further parameter switches for
 controlling model input, the data type used (single of double
 precision or explicit vectorisation are also provided.
+
+
+##### Auto-tuning for pluto
+
+Here is a script to test the best tile size to get best optimisation effect:
+```
+python tests/pluto_tile_test.py -b -s -l
```

Comment:
@mlange05 Thanks for clarifying. I'm guessing that `main` is also in `pybench.Benchmark` too then, which addresses the issues regarding `pluto_tile_test.py`.